app/background/background.ts

```diff
@@ -37,29 +46,16 @@ class BackgroundWorker {
         };
       }
       switch (request.message) {
-        case "CLEAR_DATA": {
-          this.clearData();
+        case "SET_BADGE_ZERO": {
```

Comment:
It is a generic action. Since we are using the messaging call to set it specifically to`0`, I had changed it. Using generic action names makes sense :thumbsup: will revert it